### PR TITLE
Fix #7

### LINKED ISSUE #7: Repo clean up
- [ ] I found some .DS_store files. 
- [ ] The .gitignore is not compliant with SIIP 
- [ ] Documentation is failing

### PR DESCRIPTION
**Pull request by GabrielKS**
This fixes PowerAnalytics bug #7.
---
 - Delete the offending `.DS_Store`
 - Import a good `.gitignore` from `PowerSystems`
 - Import formatter structure from `PowerSystems` as well
 - Run the formatter
 - Clean up the `docs` directory

I have been unable to reproduce the issue with the documentation build; leaving this as a draft for now until I can confirm that that is no longer an issue.

### CODE REVIEW

**Comment by GabrielKS** [comment]
@jd-lara can you check if the "Documentation is failing" issue you reported at https://github.com/NREL-Sienna/PowerAnalytics.jl/issues/7 is still present? It looks fine to me.

File: scripts/generate_config_file.py
Comment (jd-lara) [inline scripts/generate_config_file.py:1]: This file isn't needed in PowerAnalytics. Please remove

File: scripts/generate_validation_config_file.py
Comment (jd-lara) [inline scripts/generate_validation_config_file.py:1]: This file should also be removed

**Review by jd-lara — CHANGES_REQUESTED** [review]
Remove not needed files

File: scripts/generate_config_file.py
Comment (GabrielKS) [reply scripts/generate_config_file.py:1]: Done

File: scripts/generate_validation_config_file.py
Comment (GabrielKS) [reply scripts/generate_validation_config_file.py:1]: Done

**Review by jd-lara — APPROVED** [review]

**Pull request merged**